searchguard/rolesmapping.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python3
+
+import requests
+import json
+import searchguard.settings as settings
+from searchguard.exceptions import *
```

Comment:
if you are explicit about what exceptions to import instead of importing * it will be easier to spot unused code later